requirements.txt

```diff
@@ -1,8 +1,9 @@
-git+https://github.com/Rapptz/discord.py@3f06f247c039a23948e7bb0014ea31db533b4ba2#egg=discord.py
-sqlalchemy
+asyncpg
+discord.py>=1.2.5[voice]
```

Comment:
Probably should lock this down to a version since discord.py seems to have breaking changes fairly often and we're bad at fixing them in a timely manner.

Reply:
Done